assets/home/tasks-section/ready.test.js

```diff
@@ -20,7 +20,12 @@ describe( '<Ready />', () => {
 
 		const scope = nock( 'http://localhost' ).post( '/' ).reply( 200, {} );
 
-		const { container } = render( <Ready onDismiss={ onDismissMock } /> );
+		const { container } = render(
+			<Ready
+				coursePermalink="http://my-site/my-course"
+				onDismiss={ onDismissMock }
+			/>
+		);
```

Comment:
WDYT about adding a check to verify if the encoded permalink is being included in the social links?